enterprise-suite/values.yaml

```diff
@@ -122,3 +122,16 @@ prometheusVolumeSize: 256Gi
 alertmanagerVolumeSize: 32Gi
 # Grafana volume size - used for storing users, custom dashboards, and plugins.
 esGrafanaVolumeSize: 32Gi
+
+#################################################
+# Grafana configs
+#
+# Any additional configuration for Grafana, using environment variables as specified at
+# http://docs.grafana.org/installation/configuration/#using-environment-variables.
+#
+esGrafanaConfigs:
```

Comment:
I think it should be called `grafanaEnvVars` or something similar, so it's clear what they are.